lite-locals.tf

```diff
@@ -432,8 +433,8 @@ locals {
       memory = format("%dG", local.thor_worker_ram)
     }
     workerMemory = {
-      query      = format("%dG", local.thor_worker_ram)
-      thirdParty = "500M"
+      query      = format(%dG", local.workerMemory_query)
+      thirdParty = format(%dG", local.workerMemory_thirdParty)
```

Comment:
This is not the same as the original.  The hardcoded value was "500M" but your computed value is "500G" -- definitely an error or OOM condition if a Thor job starts.